editor/plugins/animation_player_editor_plugin.cpp

```diff
@@ -1573,7 +1573,7 @@ AnimationPlayerEditor::AnimationPlayerEditor(AnimationPlayerEditorPlugin *p_plug
 
 	frame = memnew(SpinBox);
 	hb->add_child(frame);
-	frame->set_custom_minimum_size(Size2(60, 0));
+	frame->set_custom_minimum_size(Size2(80, 0));
```

Comment:
Now is a good time to make the slider size set accordingly to the editor's scale factor, to handle hiDPI displays:

```suggestion
	frame->set_custom_minimum_size(Size2(80, 0) * EDSCALE);
```